packages/preset-built-in/src/plugins/features/forkTSChecker.ts

```diff
@@ -0,0 +1,28 @@
+import { IApi } from 'umi';
+
+export default (api: IApi) => {
+  api.describe({
+    key: 'enableTSChecker',
+    config: {
+      schema(joi) {
+        return joi.boolean();
```

Comment:
除非很确定只有布尔值，否则用 object，后续好扩展。

```suggestion
        return joi.object();
```